modules/button/react/README.md

```diff
@@ -825,3 +828,51 @@ Default: `undefined`
 > The icon of the button. Optional because ToolbarIconButton can also wrap a SystemIcon component.
 
 ---
+
+# ToolbarDropdownButton
+
+> Button containing an icon or custom element. Icon may be a component from
+> [`canvas-kit-react-icon`](../../icon/react) or an svg element that is used in toolbars. By
+> default, the button has a down chevron to the right indicating that it's a dropdown button.
+
+## Usage
+
+```tsx
+import * as React from 'react';
+import {ToolbarDropdownButton} from '@workday/canvas-kit-react-button';
+import {activityStreamIcon} from '@workday/canvas-system-icons-web';
+<ToolbarDropdownButton
+  icon={activityStreamIcon}
+  title="Activity Stream"
```

Comment:
Generally we suggest just aria-label (and maybe a Tooltip), don't we? The `title` tag is a little ugly :P

Reply:
ah i think this was copy pasta from icon button, I'll check those examples as well to remove title!